### PR TITLE
language: put sdk versions into package.json

### PR DESCRIPTION
**Pull request by robin-da**
The typescript library versions of our support libraries are now given
by the sdk version.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: bazel_tools/sdk_version.bzl [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    repository_ctx.file(
+        "BUILD",
+        "",
+        False,
+    )
+
+sdk_version = repository_rule(
+    implementation = _sdk_version_impl,
+    attrs = {"version": attr.label(default = "//:VERSION")},
+    local = True,
```
Comment (aherrmann-da) [inline bazel_tools/sdk_version.bzl]:
```suggestion
```
That's not needed in this case.

Comment (robin-da) [reply bazel_tools/sdk_version.bzl]: removed it in next commit.

**Comment by aherrmann-da** [comment]
> I'm wondering if we should put `0.0.0` instead of `SDK_VERSION` into the `package.json` files. This would make the local development experience nicer. Of course, there's a very small risk that we ever need to use another package at version `0.0.0`. However, I would resolve that issue when it occurs, if it ever does.

In `rules_nodejs` they use `0.0.0-PLACEHOLDER` for that purpose. We could use a similar suffix to avoid collision. Such suffixes [are allowed in semver](https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions).

**Comment by hurryabit** [comment]
I like @aherrmann's idea. Let's do that.

**Comment by robin-da** [comment]
ok, we're using `0.0.0-SDKVERSION` now in the package.json.

**Pull request merged**